lib/app-layout/index.js

```diff
@@ -52,6 +56,7 @@ export const AppLayout = ({
       <Suspense fallback={placeholder}>
         <div className="app-layout__source-column theme-color-bg theme-color-fg">
           <SearchBar noteBucket={noteBucket} onNoteOpened={onNoteOpened} />
+          {filter.length > 0 && <TagSuggestions query={filter} />}
```

Comment:
for the sake of data flow and consistency we can keep the `filter` check here but pass in `filter` to `TagSuggestions` in its own `connect()`